sdks/builds/ios.mk

```diff
@@ -301,29 +301,6 @@ $(eval $(call iOSSimulatorTemplate,sim64,x86_64))
 $(eval $(call iOSSimulatorTemplate,simtv,x86_64))
 $(eval $(call iOSSimulatorTemplate,simwatch,i386))
 
-ifndef IGNORE_PACKAGE_LLVM
-
-# Download a prebuilt llvm
-.stamp-ios-llvm-$(LLVM_HASH):
-	./download-llvm.sh $(LLVM_HASH) $(LLVM_JENKINS_LANE)
-	touch $@
-
-build-ios-llvm: .stamp-ios-llvm-$(LLVM_HASH)
-
-clean-ios-llvm: clean-llvm-llvm32 clean-llvm-llvm64
-	$(RM) -rf ../out/ios-llvm64 ../out/ios-llvm32 .stamp-ios-llvm-$(LLVM_HASH)
-
-else
-
-build-ios-llvm: package-llvm-llvm64 package-llvm-llvm32
-	ln -sf ../out/llvm-llvm32 ../out/ios-llvm32
-	ln -sf ../out/llvm-llvm64 ../out/ios-llvm64
-
```

Comment:
Please keep these dirs/symlinks, xamarin.ios depends on it. It also allows mixing the downloaded/build versions more easily during development.

Reply:
It seems that these comments were overlooked in the reviewing of this code. In order to restore XI support, the following PR has been made:

https://github.com/mono/mono/pull/10155

Reply:
Maybe put the comments in the ios.mk instead of the PR?
Otherwise requires git blame and then read the commit history for each line.